connectors/sources/s3.py

```diff
@@ -362,6 +375,47 @@ def get_default_configuration(cls):
                 "sensitive": True,
                 "type": "str",
             },
+            "s3_addressing_style": {
+                "default_value": "auto",
+                "display": "dropdown",
+                "label": "S3 Bucket Addressing Style",
+                "options": [
+                    {"label": "Auto", "value": "auto"},
+                    {"label": "Virtual", "value": "virtual"},
+                    {"label": "Path", "value": "path"},
+                ],
+                "order": 4,
+                "type": "str",
+                "value": "auto",
+                "tooltip": 'AWS users are recommended to use "Auto", S3-compatible users should refer to their provider.',
+                "ui_restrictions": ["advanced"],
+            },
+            "s3_endpoint_url": {
+                "label": "S3 Endpoint URL",
+                "order": 5,
+                "required": False,
+                "type": "str",
+                "tooltip": "AWS users are recommended to keep this blank, S3-compatible users should enter a complete URL from their provider.",
+                "ui_restrictions": ["advanced"],
+            },
+            "aws_use_fips_endpoint": {
+                "display": "toggle",
+                "label": "Use FIPS 140-3 Endpoints",
+                "order": 6,
+                "tooltip": "Not available in all regions.",
+                "type": "bool",
+                "ui_restrictions": ["advanced"],
+                "value": False,
+            },
+            "aws_use_dualstack_endpoint": {
+                "display": "toggle",
+                "label": "Use Dualstack Endpoints",
+                "order": 7,
```

Comment:
Can you go through the `order` values for the configurations after this and update them?